e2e/cypress/integration/plugins/marketplace/marketplace_spec.js

```diff
@@ -86,6 +87,53 @@ describe('Plugin Marketplace', () => {
         });
     });
 
+    describe('should render in main menu', () => {
+        afterEach(() => {
+            cy.get('#lhsHeader').should('be.visible').within(() => {
+                // # Click hamburger main menu
+                cy.get('#sidebarHeaderDropdownButton').click();
+
+                // * Dropdown menu should be visible
+                cy.get('.dropdown-menu').should('be.visible').within(() => {
+                    // * Plugin Marketplace button should be visible
+                    cy.findByText('Plugin Marketplace').should('be.visible');
+                });
+            });
+        });
+
+        it('MM-T1960 Marketplace is available when "Enable Plugins" is true', () => {
+            // # Login as sysadmin
+            cy.apiAdminLogin();
+
+            // # Enable Plugins
+            // # Enable Plugin Marketplace
+            cy.apiUpdateConfig({
+                PluginSettings: {
+                    Enable: true,
+                    EnableMarketplace: true,
+                    MarketplaceUrl: 'https://api.integrations.mattermost.com',
+                },
+            });
+            cy.visit(townsquareLink);
+        });
+
+        it('MM-T1960 Marketplace is available when "Enable Marketplace" is set to true', () => {
```

Comment:
Also could this be changed to MM-T1958 to match the test case ID?
```suggestion
        it('MM-T1958 Marketplace is available when "Enable Marketplace" is set to true', () => {
```